src/rabbit_node_monitor.erl

```diff
@@ -414,7 +414,11 @@ handle_cast({check_partial_partition, Node, Rep, NodeGUID, MyGUID, RepGUID},
                    fun () ->
                            case rpc:call(Node, rabbit, is_running, []) of
                                {badrpc, _} -> ok;
-                               _           -> cast(Rep, {partial_partition,
+                               _           ->  
+				   rabbit_log:warning("Unexpected running node:"
```

Comment:
I don't think this message is very descriptive. It lacks context entirely. What it should say (for example):

```
Received a 'DOWN' message from X but still can communicate with it
```